src/sentry_core.c

```diff
@@ -570,6 +572,24 @@ sentry__prepare_user_feedback(sentry_value_t user_feedback)
     return NULL;
 }
 
+sentry_envelope_t *
+sentry__prepare_metrics(const char *encodedMetrics)
```

Comment:
I've seen this in multiple places, and it's not super important for the first run, but we use snake-case for variables in the Native SDK, not camel-case.